src/index.html

```diff
@@ -7,6 +7,340 @@
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__rating-review">Reviews: 5</p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-label">Price:</p>
+              <h2 class="card__price-value">$2,199</h2>
+            </div>
+          </div>
+
+          <a
+            class="card__link"
+            href="#"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="Imac_image"
+            class="card__image"
+          >
+          <div class="card__info">
+            <h2 class="card__info-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <p class="card__info-code">Product code: 195434</p>
+
+            <div class="card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__rating-review">Reviews: 5</p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-label">Price:</p>
+              <h2 class="card__price-value">$2,199</h2>
+            </div>
+          </div>
+
+          <a
+            class="card__link"
+            href="#"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="Imac_image"
+            class="card__image"
+          >
+          <div class="card__info">
+            <h2 class="card__info-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <p class="card__info-code">Product code: 195434</p>
+
+            <div class="card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__rating-review">Reviews: 5</p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-label">Price:</p>
+              <h2 class="card__price-value">$2,199</h2>
+            </div>
+          </div>
+
+          <a
+            class="card__link"
+            href="#"
+            data-qa="hover"
```

Comment:
Remove all `data-qa="hover"` in all `.card` after the first.
add data-qa="card-hover" (not just hover) to the link Buy inside the first card

Reply:
Thanks, I have no idea how I missed that